### PR TITLE
[17.0][MIG] github_connector: Migration to 17.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 17.0

Please @pedrobaeza can you review it?

@Tecnativa TT51638

### CODE REVIEW

File: github_connector/models/github_repository.py [hunk 1]
```diff
@@ -108,11 +108,8 @@ def _compute_team_qty(self):
     @api.depends("name", "organization_id.github_name")
     def _compute_complete_name(self):
         for repository in self:
-            repository.complete_name = "%(login)s/%(rep_name)s" % (
-                {
-                    "login": repository.organization_id.github_name,
-                    "rep_name": repository.name or "",
-                }
+            repository.complete_name = "{}/{}".format(
```
Comment (pedrobaeza) [inline github_connector/models/github_repository.py]: Better use f-strings

**Review by pedrobaeza — APPROVED** [review]

File: github_connector/models/github_repository_branch.py [hunk 2]
```diff
@@ -179,7 +179,7 @@ def _download_code(self):
                         )
                         % (branch.local_path)
                     ) from None
-                command = ("git clone %s -b %s %s") % (
+                command = ("git clone {} -b {} {}").format(
```
Comment (carlos-lopez-tecnativa) [inline github_connector/models/github_repository_branch.py]: Could you use an `f-string`?

**Review by carlos-lopez-tecnativa — APPROVED** [review]
LGTM,  just a minor change if possible

**Pull request merged**